docs/Design/Exceptions/Exception Handling.md

```diff
@@ -0,0 +1,316 @@
+# Exception Handling
+
+This document will cover both try/catch/finally logic.
+Catch blocks detect exceptions thrown in the try block, and run the code contained in the catch block.
+Finally blocks always run.
+
+## Commands
+
+### `try start`
+
+`try start`
+
+Begins a try block.
+
+### `try end`
+
+`try end`
+
+Ends a try block.
+
+### `catch start`
+
+`catch start : exceptionType [alias]`
```

Comment:
It should be `ExceptionType` (**P**ascalCase, not **c**amelCase) since it'll be a class.